src/authz/topics.test.ts

```diff
@@ -1,41 +1,11 @@
 import * as assert from "assert";
 import sinon from "sinon";
 import { TEST_CCLOUD_KAFKA_TOPIC, TEST_LOCAL_KAFKA_TOPIC } from "../../tests/unit/testResources";
+import { createTestTopicData } from "../../tests/unit/testUtils";
 import { TopicData, TopicV3Api } from "../clients/kafkaRest";
 import * as sidecar from "../sidecar";
 import { KAFKA_TOPIC_OPERATIONS } from "./constants";
 import { fetchTopicAuthorizedOperations } from "./topics";
-import { KafkaTopicOperation } from "./types";
-
-// TODO: make this a more generic function (or `TEST_TOPIC_DATA` constant) that can be used in other
-// tests if we need to start using it more
-function createTestTopicData(
```

Comment:
Relocated to `../../tests/unit/testUtils` per this TODO because needed to use in a separate test.